### ISSUE TITLE
Can't install tensorflow-text

### ISSUE THREAD

**Issue by georgealexandruvlad** [tensorflow/tensorflow#30998]

**Comment by georgealexandruvlad** [comment]
I can't install tensorflow-text via pip as it is shown in the tf2.0 tutorial pages [https://www.tensorflow.org/beta/tutorials/tensorflow_text/intro](url). I tried using -q or -U flag (as it is specified in tensorflow-text github page) but without any success. I made an environment in anaconda and installed tf2.0. The evironment has python 3.7.3 but I tried with other versions too. I checked pip version to be higher than 19.1 as the tutorial mentions it. Bellow is the error I get.

![image](https://user-images.githubusercontent.com/44173617/61817722-db87ab00-ae57-11e9-979e-7d84be100cce.png)

On google colab I can install it (checking that there's nothing related to the package availability).
